### PR TITLE
Mkdocs style code blocks

### PR DESCRIPTION
**Pull request by csantanapr**
- helper script to run container
- made copy clipboard color more visible
- hide copy clipboard when not need it
- fix identation in bash block blocks

### CODE REVIEW

File: dev/run.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+NAME=${1:-mkdocs}
+PORT=${2:-8000}
+IMAGE=${3:-mkdocs}
+
+docker run --name "${NAME}" -d -p "${PORT}:${PORT}" -v "${PWD}:/site" mkdocs serve -f /site/mkdocs.yml --livereload -a "0.0.0.0:${PORT}"
+echo "Dev environment running with live reloading enabled. Open http://localhost:${PORT} to see the site"
```
Comment (julz) [inline dev/run.sh]: 🎉

File: docs/getting-started/first-traffic-split.md [hunk 2]
```diff
@@ -119,10 +119,11 @@ By default, when Knative creates a brand new Service it directs 100% of traffic
     ```
 
 Now when we curl our Knative Service URL...
-```bash
-      curl <service-url>
+```{ .optional-language-as-class .no-copy }
```
Comment (julz) [inline docs/getting-started/first-traffic-split.md]: this should be:

```suggestion
```{ .bash .no-copy }
```

(same elsewhere obviously with `.optional-language-as-class` being whatever language the fenced code block should be highlighted in)

File: docs/stylesheets/extra.css [hunk 3]
```diff
@@ -2,3 +2,7 @@
   --md-primary-fg-color:       #1B8DE3;
   --home-color:                #0071c7;
 }
+.md-clipboard {
+  color: var(--md-primary-fg-color);
```
Comment (julz) [inline docs/stylesheets/extra.css:6]: neat :).

File: docs/stylesheets/extra.css [hunk 4]
```diff
@@ -2,3 +2,7 @@
...
+}
+.highlight.no-copy .md-clipboard { display: none; }
```
Comment (julz) [inline docs/stylesheets/extra.css]: nit: can we add a newline at the end of this?

File: dev/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,2 @@
+FROM quay.io/cloudnativetoolkit/mkdocs
```
Comment (julz) [inline dev/Dockerfile]: out of interest, why cant we FROM the regular material image here?

**Review by julz — COMMENTED** [review]
awesome! spotted a couple nits:

File: dev/Dockerfile (hunk 5 above)
Comment (csantanapr) [reply dev/Dockerfile]: yes we can use the material one

File: dev/run.sh (hunk 1 above)
Comment (csantanapr) [reply dev/run.sh]: I added a new line to follow the logs

File: dev/Dockerfile (hunk 5 above)
Comment (csantanapr) [reply dev/Dockerfile]: yes we can use the material one

**Pull request merged**